colosseum/fonts.py

```diff
@@ -0,0 +1,95 @@
+from .constants import (FONT_FAMILY_CHOICES, FONT_SIZE_CHOICES,
+                        FONT_STYLE_CHOICES, FONT_VARIANT_CHOICES,
+                        FONT_WEIGHT_CHOICES, INHERIT, INITIAL_FONT_VALUES,
+                        NORMAL, SYSTEM_FONT_KEYWORDS)
+from .validators import ValidationError
+
+
+def get_system_font(keyword):
+    """Return a font object from given system font keyword."""
+    if keyword in SYSTEM_FONT_KEYWORDS:
+        return '"Arial Black"'
+        # Get the system font
+    return None
+
+
+def construct_font_property(font):
+    """Construct font property string from a dictionary of font properties."""
+    if isinstance(font['font_family'], list):
+        font['font_family'] = ', '.join(font['font_family'])
+
+    return ('{font_style} {font_variant} {font_weight} '
+            '{font_size}/{line_height} {font_family}').format(**font)
+
+
+def parse_font_property_part(value, font):
+    """Parse font shorthand property part for known properties."""
+    if value != NORMAL:
+        for property_name, choices in {'font_variant': FONT_VARIANT_CHOICES,
+                                       'font_weight': FONT_WEIGHT_CHOICES,
+                                       'font_style': FONT_STYLE_CHOICES}.items():
+            try:
+                value = choices.validate(value)
+                font[property_name] = value
+                return font
+            except (ValidationError, ValueError):
+                pass
+
+        # Maybe it is a font size
+        if '/' in value:
+            font['font_size'], font['line_height'] = value.split('/')
+            return font
+        else:
+            try:
+                FONT_SIZE_CHOICES.validate(value)
+                font['font_size'] = value
+                return font
+            except ValueError:
+                pass
+
+        raise ValidationError
```

Comment:
Should raise an instance, not the class.

Reply:
Fixed